qos-monitor/src/main/java/eu/arrowhead/core/qos/service/ping/monitor/PingEventBufferCleaner.java

```diff
@@ -0,0 +1,47 @@
+package eu.arrowhead.core.qos.service.ping.monitor;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+import javax.annotation.Resource;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.stereotype.Service;
+
+import eu.arrowhead.core.qos.QosMonitorConstants;
+
+@Service
+public class PingEventBufferCleaner {
+	//=================================================================================================
+	// members
+
+	private final long MAX_TIME_BEFORE_CLEAR = 1000 * 60 * 10 * 2;
```

Comment:
maybe a comment what these magic numbers mean. 1000 because of conversion from seconds to milliseconds, 60 for a minute. But I don't know what's the meaning of 10 and 2.

Reply:
done